### PR TITLE
Gp19 91 Prefilled edit form

### PR DESCRIPTION
**Pull request by karenzamani**

### CODE REVIEW

File: src/User/ProfilePage.jsx [hunk 1]
```diff
@@ -32,16 +34,18 @@ const ProfilePage = ({ error, isLoading, user }) => {
...
-      <StyledLink className="editLink" to={`/users/edit/${id}`}>Edit</StyledLink>        
+      <StyledLink className="editLink" to={`/users/${id}/edit`}>Edit</StyledLink>        
       <StyledCardHeading className="name">
         {`Hello, ${firstName} ${lastName}!`}
       </StyledCardHeading>
       <p className="username">{`Username: ${username}`}</p>
       <p className="email">{`Email: ${email}`}</p>
       <p className="occupation">{`Occupation: ${occupation}`}</p>
-    </StyledCard>
+      </StyledCard>
```
Comment (RaMorey) [inline src/User/ProfilePage.jsx]: Could probably just format this slightly nicer for readability. Shift card in 1 tab and Styled profile back one so it's indented once per level

File: src/User/UpdateProfile.jsx [hunk 2]
```diff
@@ -1,17 +1,82 @@
...
+        setUpdatedUser({ ...updatedUser, [name]: value });
+    }
+    
+    const submitForm = async () => {
+        try {
+          await updateUserById(id, updatedUser, sessionToken.token);
+          setUser(updatedUser);
+          history.push(`/users/${id}`);
+        } catch (error) {
+          console.log(error);
```
Comment (RaMorey) [inline src/User/UpdateProfile.jsx]: If you're only logging the error you can get rid of the try catch

File: src/User/UpdateProfile.jsx [hunk 3]
```diff
@@ -1,17 +1,82 @@
...
+        }
+    };
+
+    return (
+        <StyledForm>
```
Comment (RaMorey) [inline src/User/UpdateProfile.jsx]: I think you need to fix your prettier settings, the indentation and formatting is quite inconsistent across different files

File: src/User/UserContainer.jsx [hunk 4]
```diff
@@ -14,27 +15,44 @@ const UserContainer = ({ token }) => {
     setIsLoading(true);
 
     try {
-      const user = await getUserById(userId, token);
+      const user = await getUserById(userId, sessionToken.token);
       setUser(user);
+      setUpdatedUser({
```
Comment (RaMorey) [inline src/User/UserContainer.jsx]: Would you be able to explain what this line and the one above it are doing? I'm a little confused

File: src/User/UserContainer.jsx [hunk 5]
```diff
@@ -14,27 +15,44 @@ const UserContainer = ({ token }) => {
...
+        ...user
+      });
     } catch (error) {
       setError(true);
     }
     setIsLoading(false);
   };
 
   useEffect(() => {
+    console.log(id);
```
Comment (RaMorey) [inline src/User/UserContainer.jsx]: Can remove this now

File: src/User/UserContainer.jsx (hunk 4 above)
Comment (RaMorey) [reply src/User/UserContainer.jsx]: I think after reading it a bit more, you should consider moving the updatedUser state into the UpdateProfile component

File: src/api.js [hunk 6]
```diff
@@ -61,9 +61,11 @@ export const loginUser = async user => {
 };
 
 export const getUserById = async (id, token) => {
+  console.log(token)
```
Comment (RaMorey) [inline src/api.js]: log log

**Comment by RaMorey** [comment]
Also you probably want to add in some tests for the updateProfile component

File: src/api.js (hunk 6 above)
Comment (karenzamani) [reply src/api.js]: log

**Pull request closed without merging**